src/ci/docker/dist-various-2/build-wasi-toolchain.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -ex
+
+curl http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-14.04.tar.xz | \
```

Comment:
Might make sense to use latest LTS (18.04) instead

Reply:
We will eventually! For now though the rest of the image here is 17.10, but when that upgrades we can upgrade this